test/integration_test/stork-test-pod.yaml

```diff
@@ -70,6 +70,8 @@ spec:
       value: username
     - name: TORPEDO_SSH_PASSWORD
       value: password
+    - name: LS_CONTROLLERS
```

Comment:
Don't you have to parameterize `-volume-driver` on line 59?

Reply:
What do you mean? I've changed it back to `pxd` for the PR. The `LS_CONTROLLERS` is required so that LINSTOR can find its controller pod

Reply:
I can't add a comment on line 59, so added it here. `LS_CONTROLLERS` is fine.

But, on line 59 the driver name is being passed into the test and I thought you needed to update that too. I see that is being updated by test-deply.sh though, so no change needed there